pkg/virt-operator/resource/generate/rbac/cluster.go

```diff
@@ -141,6 +141,7 @@ func newAdminClusterRole() *rbacv1.ClusterRole {
 					"virtualmachineinstances/console",
 					"virtualmachineinstances/vnc",
 					"virtualmachineinstances/vnc/screenshot",
+					"virtualmachineinstances/vsock",
```

Comment:
I think at least for now, for security reasons we agreed to not add this permission to namespace-admin and namespace-edit, at leas for now reserving it completely for internal use.

Reply:
Done